src/main/java/io/minio/MinioClient.java

```diff
@@ -177,7 +177,7 @@ public MinioClient(String endpoint) throws InvalidEndpointException, InvalidPort
    * @see #MinioClient(String endpoint, String accessKey, String secretKey, boolean secure)
    * @see #MinioClient(String endpoint, int port, String accessKey, String secretKey, boolean secure)
    */
-  public MinioClient(URL url) throws NullPointerException, InvalidEndpointException, InvalidPortException {
+  public MinioClient(URL url) throws InvalidEndpointException, InvalidPortException {
```

Comment:
Please clarify removing `NullPointerException` in under https://dev.eclipse.org/sonar/rules/show/squid:RedundantThrowsDeclarationCheck